src/app/pages/OpenWalletPage/Features/FromBleLedger/__tests__/index.test.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react'
+import { render } from '@testing-library/react'
+import { FromBleLedger } from '..'
+
+jest.mock('react-redux', () => ({
+  useSelector: jest.fn(),
+  useDispatch: jest.fn(),
+}))
+
+describe('<FromLedger />', () => {
+  it('should render component', () => {
+    const { container } = render(<FromBleLedger />)
+
+    expect(container).toMatchSnapshot()
```

Comment:
This test creates 2xx lines where non of a line cover logic and just checks i18n keys. If we are interested in labels we can clear PR by using screen API. What test should do is rather test conditions and redux actions if needed.